content/articles/introduction-to-udoo-x86-ii-ultra/index.md

```diff
@@ -0,0 +1,106 @@
+
+The embedded Arduino Leonardo 101 board comes with a six-axis accelerometer and a gyroscope built in. The customizability offered by the Arduino board makes this session board controller a very resourceful/powerful tool for casual uses.
+
+Udoo X86 II Ultra is based on the new generation Quad Core 64-bit personal computer with the intel's X86 processor architecture.
+
+With an energy consumption of about 5 to 6 Watts, the gigantic processor is concentrated in nanometers. The power can be provided via the VIN pins usually found on the board's head.
+
+### Pros
+Udoo X86 II Ultra session board has the following advantages:
+
+#### Enormous RAM
+Udoo X86 II Ultra comes with an 8GB built in RAM. The advanced version has 4GB of RAM while the basic one has 2GB of RAM. This makes the Udoo X86 II Ultra more powerful than most of the session board controllers on the market.
+
+#### Efficiency
+Udoo X86 II Ultra is capable of running software meant for a normal personal computer. This is made possible by a 2.56GHz quad-core intel CPU and an 8GB RAM.
+
+#### Availability of freely useful guides and community
+Through a helpful community even not all guides are available online.
+
+###	Lower energy consumptions
+The Arduino Leonardo 101 board requires less than 11 Watts of energy which makes it very energy efficient.
+
+#### Large storage capacity
+Udoo X86 II Ultra box comes with a 32GB embedded multimedia card. However, users might decide to upgrade the storage to fit their desire. Though expensive, an SSD card is usually the best option. Additionally, a micro-SD card can also be used as a storage media.
+
+#### X86-compatible architecture
+Udoo X86 II Ultra supports most operating systems including Linux, Windows and Android. For Linux and Android, the internal storage can be used as a read only memory with additional SATA `HDD/SSD/eMMC` drive used as /tmp and /home directories.
+
+Udoo II X86 Ultra is also used a gaming emulator as it supports most retro gaming emulators available for a PC. It can also emulate new consoles like play-station II.
+
+#### Decent GPIO support
+The Udoo x86 Ultra comes with 28 `GPIO` ports in total which may seem few, but considering that it has an Arduino 101 embedded on it, this is a decent number.
+
+The Arduino board has an in built functionality for what some pins can be used for (like a six-axis accelerometer and gyroscope) and it adds 12 additional GPIO pins. Several key communication protocols such as the two IIC, two UART, LPC and SDIO are also supported.
+
+####  Easy and straightforward set up
+It’s absolutely straightforward to set up. The installation guidelines for the supported operating systems are provided in hard-copy and video form.
+
+#### Arduino compatible
+Udoo x86 II Ultra comes with an embedded Arduino 101, with all upgrades enabled (including 6-axis accelerometer, gyroscope and Bluetooth).
```

Comment:
You have already mentioned Arduino compatibility in the previous parts. Consider removing this section.

Reply:
Already removed.